test/migrators/module/indirect_configuration.hrx

```diff
@@ -0,0 +1,22 @@
+<==> arguments
+--migrate-deps
+
+<==> input/entrypoint.scss
+$config: red;
+@import "direct";
+
+<==> input/_direct.scss
+@import "indirect";
+
+<==> input/_indirect.scss
+$config: green !default;
+
+<==> output/entrypoint.scss
+$config: red;
```

Comment:
We should get rid of the original variable definitions if they're used for configuration, and replace later uses with `$direct.config`.